Makefile

```diff
@@ -294,20 +301,30 @@ docs_serve:
 # Test #
 ########
 
-subsystem-run: test subsystem-clean
+subsystem-run: | test-rest-api subsystem-clean enable-kube-api-for-subsystem test-kube-api subsystem-clean
```

Comment:
Why do we need to clean after each test run? 
Actually, it looks like it does nothing.. Maybe that target can be completely removed